modules/button/react/spec/IconButton.spec.tsx

```diff
@@ -22,6 +23,32 @@ describe('Icon Button', () => {
     component.unmount();
   });
 
+  it('should have an identifier class', () => {
+    const {getByTestId} = render(
+      <IconButton data-testid={iconButtonIdentifier} aria-label="Activity Stream">
```

Comment:
Nit: it's a little bit confusing at first glance to use the identifier for both the test ID and the classname. Any reason we can't drop the test ID and use `getByRole('button')`?

Reply:
Because I didn't think of that... 🤦‍♂ 😅